src/main/java/gov/nasa/cumulus/SenderFactory.java

```diff
@@ -0,0 +1,16 @@
+package gov.nasa.cumulus;
+
+/**
+ * Factory class to return Sender based on type
+ */
+public class SenderFactory {
+    public static Sender getSender(String region, String type) {
+        if ("kinesis".equalsIgnoreCase(type)) {
+            return new KinesisSender(region);
+        } else if ("sns".equalsIgnoreCase(type)) {
+            return new SNSSender(region);
+        } else {
```

Comment:
```suggestion

/**
 * Factory class to return Sender based on type
 */
public class SenderFactory {
    public static final string METHOD_KINESIS = "kinesis"; 
    public static final string METHOD_SNS = "sns";   
    
    public static Sender getSender(String region, String type) {
        if (METHOD_KINESIS.equalsIgnoreCase(type)) {
            return new KinesisSender(region);
        } else if (METHOD_SNS.equalsIgnoreCase(type)) {
            return new SNSSender(region);
        } else {
```

Reply:
might even be pulled outside senderFactory, but at least it's available outside if needed.